root/static/scripts/relationship-editor/common/entity.js

```diff
@@ -29,6 +29,8 @@ function getDirection(relationship, source) {
   if (source === entities[1]) {
     return 'backward';
   }
+
+  throw 'source should be in the entities array (which has length 2)';
```

Comment:
shortening this to something very minimal should be fine, since it should never happen (and these strings can't be minified for production, so having long strings in our js bundles that aren't used isn't ideal)
```suggestion
  throw 'source not in the entities array';
```